easybuild/easyconfigs/n/nvtop/nvtop-1.0.0-fosscuda-2018b.eb

```diff
@@ -0,0 +1,32 @@
+easyblock = 'CMakeMake'
+
+name = 'nvtop'
+version = '1.0.0'
+
+homepage = 'https://github.com/Syllo/nvtop'
+description = 'htop-like GPU usage monitor'
+
+toolchain = {'name': 'fosscuda', 'version': '2018b'}
+
+source_urls = ['https://github.com/Syllo/nvtop/archive/']
+sources = ['%(version)s.tar.gz']
+checksums = ['de3d7b6a889f886f3bc28c15c3c34cfb6fcb7cb7ebc9a0cc36c3c77d837029b1']
+
+configopts = '-DNVML_INCLUDE_DIRS=$EBROOTCUDA/include -DNVML_LIBRARIES=$EBROOTCUDA/lib64/stubs/libnvidia-ml.so'
```

Comment:
@adiemu5 Please move this down below `separate_build_dir = True` (that's just a style issue, it doesn't affect the installation).

One thing I'm not sure about is whether providing the `stubs` library is the right way to go here... Maybe @akesandgren knows?

Reply:
Still being new to github, I assume by "this" you mean line 15 (configopts) should go below "separate_build_dir".  Right?  (Or was all of line 12-15 supposed to move, which seems less likely)

As to pointing CMake at the 'stubs' library, that's the only way I could figure out to get it to successfully build on a system that didn't have a system-install of CUDA on it.  I tried to figure out the best way to do this by referencing other easyconfigs that use NVML, but GROMACS was the only one I could find and I honestly can't figure out how it avoids this problem.

Reply:
Went ahead and moved configopts like I think you've requested.

Any word on the 'stubs' question?

Reply:
Using the stubs is safe unless you're using RPATH, because then it will be pointed to the stubs even when it is used.

But it's always better to have the real nvidia-dev packages installed.

Reply:
Ultimately, I used the stubs so that it can build anywhere, as long as fosscuda-2018b is present and loaded, regardless of whether any NVIDIA hardware or drivers are present..  However, since this package only runs on systems with NVIDIA GPUs, requiring the dev packages seems reasonable too, but I don't know how I would do that in the easyconfig.

Reply:
@adiemu5 Sorry for not getting back to you on this, I was indeed suggesting to only move the `configopts` line, as you did.